Bryan-Cee/Fast-Food-Fast
# #160089190 Finish the admin track order page

#### What does this PR do?
Creates the Administrator page for tracking the state of orders

#### Description of Task to be completed?

- Design and style the admin tracking page 
- Add a Marks as done button

#### How should this be manually tested?

- Clone the repo and `cd` into `UI/pages/admin`
- Open `orders.html` with a browser
- Clicking accepted to enable marking as complete button appear

#### What are the relevant pivotal tracker stories?
[#160089190](https://www.pivotaltracker.com/story/show/160089190)

#### Screenshots
![ordersfinal](https://user-images.githubusercontent.com/19145546/44900639-c7056600-ad0d-11e8-9418-e2c0d42af9f3.png)
![completed](https://user-images.githubusercontent.com/19145546/45435207-bfa06e00-b6b8-11e8-9fb4-c33ea11f969e.PNG)
